src/test/scala/beam/router/PhyssymXmlToOsmConverter.scala

```diff
@@ -288,6 +290,11 @@ object PhyssymXmlToOsmConverter extends StrictLogging {
     osmWay
   }
 
+  private def metersPerSecondToKilometersPerHour(speed: String): String = {
+    val output = try { speed.toDouble * 3.6 } catch { case _: Throwable => DEFAULT_LINK_SPEED_KPH }
```

Comment:
Since you aren't logging the exception, you can do:

`scala.util.Try(speed.toDouble * 3.6).getOrElse(DEFAULT_LINK_SPEED_KPH).toString`